README.md

```diff
@@ -21,10 +21,13 @@ The primary interface is as a command line tool, usually run like so:
 ```sh
 $ codemod --plugin transform-module-name \
   path/to/file.js \
-  another/file.js
+  another/file.js \
+  a/directory
 ```
 
-This will re-write the files `path/to/file.js` and `another/file.js` by transforming them with the babel plugin `transform-module-name`. Multiple plugins may be specified, and multiple files or directories may be re-written at once.
+This will re-write the files `path/to/file.js` and `another/file.js` any supported files found in `a/directory` by transforming them with the babel plugin `transform-module-name`. Multiple plugins may be specified, and multiple files or directories may be re-written at once.
```

Comment:
typo: "any supported files" -> "and any ..." or switch to oxford comma: "`path/to/file.js`, `another/file.js`, and supported files found in `a/directory`"

Reply:
Fixed.